devserver/devserver/BUILD.bazel

```diff
@@ -3,6 +3,9 @@ load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
 go_library(
     name = "go_default_library",
     srcs = ["devserver.go"],
+    deps = [
+        "@io_bazel_rules_go//go/tools/bazel:go_default_library",
```

Comment:
This change seems to be causing all buildkite CI to fail with the following error:
```
ERROR: D:/b/bk-worker-windows-java8-hdpm/bazel/rules-typescript-typescript/devserver/devserver/BUILD.bazel:3:1: no such package '@com_github_bazelbuild_rules_go//go/tools/bazel': The repository could not be resolved and referenced by '//devserver/devserver:go_default_library'
```

Not really sure why it doesn't affect the CircleCI setup.